src/main/resources/static/api.yml

```diff
@@ -1191,6 +1191,39 @@ paths:
                 $ref: '#/components/schemas/Problem'
         500:
           $ref: '#/components/responses/500Response'
+  /people/{crn}/oasys/selection:
+    get:
+      tags:
+        - OASys
+      summary: Returns the importable sections of OASys, grouped by whether they are linked to harm/reoffending
```

Comment:
I think we need a new summary now? Something like "Returns the importable sections of OASys including details of links to harm and reoffending"?